### PR TITLE
zh-translation: faq general what does istio mean

### PR DESCRIPTION
**Pull request by helight**
zh translation for /faq/general/what-does-istio-mean.md
ref: https://github.com/servicemesher/istio-official-translation/issues/1406

### CODE REVIEW

File: content/zh/faq/general/what-does-istio-mean.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: What does the word 'Istio' mean?
+title: 'Istio' 这个词是什么意思呢？
 weight: 160
 ---
 
-It's the Greek word for 'sail'.
+这是 'sail' 这个词的 Greek 说法。
```
Comment (ilylia) [inline content/zh/faq/general/what-does-istio-mean.md]:
```suggestion
它是希腊语中的 'sail'。
```

File: content/zh/faq/general/what-does-istio-mean.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: What does the word 'Istio' mean?
+title: 'Istio' 这个词是什么意思呢？
```
Comment (ilylia) [inline content/zh/faq/general/what-does-istio-mean.md]:
```suggestion
title: 'Istio' 这个词是什么意思？
```

File: content/zh/faq/general/what-does-istio-mean.md (hunk 1 above)
Comment (helight) [reply content/zh/faq/general/what-does-istio-mean.md]: accept

Comment (helight) [reply content/zh/faq/general/what-does-istio-mean.md]: accept

File: content/zh/faq/general/what-does-istio-mean.md (hunk 2 above)
Comment (helight) [reply content/zh/faq/general/what-does-istio-mean.md]: 这个问题不大吧

Comment (ilylia) [reply content/zh/faq/general/what-does-istio-mean.md]: 问题不大，感觉不要呢跟整体风格比较搭

Comment (helight) [reply content/zh/faq/general/what-does-istio-mean.md]: ok,accept

File: content/zh/faq/general/what-does-istio-mean.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: What does the word 'Istio' mean?
+title: 'Istio' 这个词是什么意思？
```
Comment (ilylia) [inline content/zh/faq/general/what-does-istio-mean.md]:
```suggestion
title: “Istio” 这个词是什么意思？
```

File: content/zh/faq/general/what-does-istio-mean.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
 weight: 160
 ---
 
-It's the Greek word for 'sail'.
+它是希腊语中的 'sail'。
```
Comment (ilylia) [inline content/zh/faq/general/what-does-istio-mean.md]:
```suggestion
它是希腊语中的 “sail”。
```

File: content/zh/faq/general/what-does-istio-mean.md (hunk 3 above)
Comment (helight) [reply content/zh/faq/general/what-does-istio-mean.md]: got that

File: content/zh/faq/general/what-does-istio-mean.md (hunk 4 above)
Comment (helight) [reply content/zh/faq/general/what-does-istio-mean.md]: got that

**Pull request merged**